src/components/TokenIcon.tsx

```diff
@@ -63,10 +63,10 @@ interface Props {
 
 export default function TokenIcon({
   token,
-  viewStyle,
+  viewStyle = styles.viewStyle,
```

Comment:
Instead of setting the default for styles like this, I think it's cleaner to set the default in the view itself and pass the viewStyle props afterward. This way, if I need to change just the border color, I don't also need to pass the other default styles.


```TypeScript
// Example Component
function SampleIcon({
  viewStyle,
}: Props) {
  return <View style={[style.default, viewStyle]) />
}

const styles = StyleSheet.create({
  default: { 
    borderColor: colors.black, 
    borderWidth: 1, 
    borderRadius: 20 
  }
})

// Changing just the border color
<SampleIcon viewStyle={{ borderColor: 'red' }} />

Reply:
addressed in [22a8a37](https://github.com/mobilestack-xyz/mobilestack-mento/pull/9/commits/22a8a3781665e95cdd242583b0de248dabcdf995)